src/resolve/resolved-bus.c

```diff
@@ -29,6 +29,21 @@
 
 BUS_DEFINE_PROPERTY_GET_ENUM(bus_property_get_resolve_support, resolve_support, ResolveSupport);
 
+static int dns_query_new_for_bus(
+                Manager *m,
+                DnsQuery **ret,
+                DnsQuestion *question_utf8,
+                DnsQuestion *question_idna,
+                DnsPacket *question_bypass,
+                int ifindex,
+                uint64_t flags,
+                sd_bus_error *error) {
+        int r = dns_query_new(m, ret, question_utf8, question_idna, question_bypass, ifindex, flags);
```

Comment:
Please insert an empty line after the function arguments, and declare `r` separately.
```c
                sd_bus_error *error) {

        int r;

        r = dns_query_new(...);
        if (r == -ENOANO)
```